src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
```

Comment:
Creating a new instance of `Random` every time the method is called is inefficient. Consider creating a single instance of `Random` as a class-level (private) field and reusing it.